Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -439,6 +439,61 @@ public IEnumerator TestDestroy()
             Assert.AreEqual(propBlock.GetColor("_Color"), FocusColor);
         }
 
+        [UnityTest]
```

Comment:
FYI, #6104 will have impacts on this
1) SetDisabled is now deprecated, use IsEnabled = false property
2) There is a somewhat related TestDisableState and TestSelectGlobalInputOnDisabled in that PR but they are both slightly different to this test so we still need this test